published/external/xdp/status.h

```diff
@@ -25,11 +25,14 @@ typedef NTSTATUS XDP_STATUS;
 
 typedef HRESULT XDP_STATUS;
 
+#define SUCCESS_HRESULT_FROM_WIN32(x) \
```

Comment:
We can't do this, because other code does `HRESULT_FROM_WIN32(GetLastError())` and would produce inconsistent results. What are you trying to achieve here?

Reply:
I just copied from MsQuic. trying to resolve implicit cast. I'm not sure how to resolve other than this

Reply:
I think your original solution was fine, other than a comment was needed.